src/Tribe/JSON_LD/Order.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+/**
+ * A JSON-LD class to hook and change how Orders work on Events
+ * @todo rework this class to make it standalone from The Events Calendar
+ */
+class Tribe__Tickets__JSON_LD__Order {
+
+	/**
+	 * Get (and instantiate, if necessary) the instance of the class
+	 *
+	 * @static
+	 * @return self
+	 *
+	 */
+	public static function instance() {
+		static $instance;
+
+		if ( ! $instance instanceof self ) {
+			$instance = new self;
+		}
+
+		return $instance;
+	}
+
+	/**
+	 * Value indicating low stock availability for a specific ticket.
+	 *
+	 * This can be overridden with the tribe_tickets_json_ld_low_inventory_level filter.
+	 *
+	 * @var int
+	 */
+	protected $low_stock = 5;
+
+	/**
+	 * Setup Google Event Data for tickets.
+	 */
+	public function hook() {
+		add_filter( 'tribe_json_ld_event_object', array( $this, 'add_tickets_data' ) );
+	}
+
+	/**
+	 * Used to setup variables on this class
+	 */
+	public function __construct() {
+		/**
+		 * Allow users to change the Low inventory mark
+		 * This is the old filter
+		 *
+		 * @todo remove on 4.4
+		 * @deprecated
+		 * @var int
+		 */
+		$this->low_stock = apply_filters( 'tribe_events_tickets_google_low_inventory_level', $this->low_stock );
+
+		/**
+		 * Allow users to change the Low inventory mark
+		 * @var int
+		 */
+		$this->low_stock = apply_filters( 'tribe_tickets_json_ld_low_inventory_level', $this->low_stock );
+	}
+
+	/**
+	 * Adds the tickets data to the event Object
+	 *
+	 * @param array $data
+	 * @param array $args
+	 * @param WP_Post $post
+	 *
+	 * @return array
+	 */
+	public function add_tickets_data( $data, $args, $post ) {
```

Comment:
For backwards compatibility, should this method be renamed to `add_ticket_data` (to match the definition in the now deprecated class)?